### ISSUE TITLE
There is no support for additional drivers to take a safety test

### ISSUE THREAD

**Issue by aavtamon** [aavtamon/boating#42]
There is no support for additional drivers to take a safety test - only the primary renter has the link, and that link is attached to his DL

**Comment by chuckhardt** [comment]
????   You want to add a provision for additional test takers?
Chuck Hardt
chuckhardt@yahoo.com 

    On Tuesday, March 20, 2018, 4:46:59 PM EDT, aavtamon <notifications@github.com> wrote:  
 
 
There is no support for additional drivers to take a safety test - only the primary renter has the link, and that link is attached to his DL

**Comment by aavtamon** [comment]
Yes - we need this if someone want to have other people driving the boat.
This is a normal practice - multiple people may want to share the steering
wheel

**Comment by chuckhardt** [comment]
ok, same test, different person, correct.  We do not need a different test.  At most, randomize the order of the questions.
Chuck Hardt
chuckhardt@yahoo.com 

    On Tuesday, March 20, 2018, 9:07:37 PM EDT, aavtamon <notifications@github.com> wrote:  
 
 Yes - we need this if someone want to have other people driving the boat.
This is a normal practice - multiple people may want to share the steering
wheel
